### ISSUE TITLE
Text selectable not working on android and ios

### ISSUE THREAD

**Issue by Fulanko** [microsoft/reactxp#1197]
I can't get RX.Text to be selectable on native platforms.
`<RX.Text selectable={true}>Cannot select this text</RX.Text>`

It works fine on web.
reactxp: 2.0.0
react-native: 0.61.5

**Comment by fbartho** [comment]
@fulanko is this still a concern? Looking at the Android implementation of [RX.Text](https://github.com/microsoft/reactxp/blob/master/src/android/Text.tsx) I think this feature isn't implemented on `react-native` mobile. And the documentation suggests that this "affects mouse and copy operations".

This might be a feature that was added later to react-native.

**Comment by Fulanko** [comment]
You are right. This was not originally implemented in react-native, so it's not a bug in reactxp.

**Comment by fbartho** [comment]
Please let me know if you’re interested in contributing this @Fulanko?
